doc/source/WorkingWithThePythonScriptingAPI/TutorialStartupConfig1/index.md

```diff
@@ -131,31 +131,31 @@ We wrap up the config by adding our function to an event signal that fires when
 Notice the use of the `application` variable. This is a special variable that refers to the startup config's parent application.
 
 
-## Testing the global context variable ##
+## Testing the global Context Variable ##
 
-Now we can test the startup config in a live graph. If you haven't already, save `customVariables.py`, then launch a new instance of Gaffer. In the empty graph, take a look at the global context variables found in the _Variables_ tab of the graph's settings (_File_ > _Settings_). You should see the new `project:resources` variable pointing to the correct path.
+Now we can test the startup config in a live graph. If you haven't already, save `customVariables.py`, then launch a new instance of Gaffer. In the empty graph, take a look at the global Context Variables found in the _Variables_ tab of the graph's settings (_File_ > _Settings_). You should see the new `project:resources` variable pointing to the correct path.
 
-![](images/tutorialSettingsWindowCustomContextVariable.png "The custom global context variable in the Settings window")
+![](images/tutorialSettingsWindowCustomContextVariable.png "The custom global Context Variable in the Settings window")
 
-Success! Now every node graph you open or create will have the `project:resources` global context variable added to it. Try using the variable to load Gaffy's scene cache:
+Success! Now every node graph you open or create will have the `project:resources` global Context Variable added to it. Try using the variable to load Gaffy's scene cache:
 
 1. Create a SceneReader node.
-2. In the _Node Editor_, set the _File Name_ plug to `${project:resources}/gafferBot/caches/gafferBot.scc`.
+2. In the Node Editor, set the File Name plug to `${project:resources}/gafferBot/caches/gafferBot.scc`.
 
 If all went well, Gaffy's geometry cache should have loaded in the graph.
 
 ![](images/tutorialVariableSubstitutionTest.png "Successfully reading Gaffy using variable substitution in a string")
 
-As mentioned earlier, if we wanted to, we could make the path more granular, perhaps by assigning a global context variable to each its child directories.
+As mentioned earlier, if we wanted to, we could make the path more granular, perhaps by assigning a global Context Variable to each its child directories.
 
 
-## Environment variables and context variables ##
+## Environment variables and Context Variables ##
```

Comment:
If its capital V for context variables, should it be for environment variables too?

Reply:
I was in fact contemplating just "Context variable", but we also need to be able to distinguish them from Context objects, so I consider them both discrete "things". And for comparison, Katana formats theirs as "Graph State Variables".

Environment variables are universal enough that they've become generic, and thus lower case.